Resources/Private/Fusion/Backend/Component/Scripts.fusion

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+include: resource://Neos.Fusion/Private/Fusion/Root.fusion
+
+/** @internal */
+prototype(Neos.Neos.Ui:Component.Scripts) < prototype(Neos.Fusion:Value) {
+    value = ${scripts}
```

Comment:
i know this is just a draft, but i wanted to point out that this context variable `scripts` is super sneaky and unexpected here ^^